README.md

```diff
@@ -69,17 +69,17 @@ Ready to use binaries are available [from GitHub](https://github.com/green-code-
 | 1.4.+          | 9.4.+ LTS to 10.4.1 | 11 / 17      |
 
 > Compatibility table of versions lower than 1.4.+ are available from the
-> main [ecoCode repository](https://github.com/green-code-initiative/ecoCode#-plugins-version-compatibility).
+> main [creedengo repository](https://github.com/green-code-initiative/creedengo#-plugins-version-compatibility).
 
 🤝 Contribution
 ---------------
 
-check [ecoCode repository](https://github.com/green-code-initiative/ecoCode#-contribution)
+check [creedengo repository](https://github.com/green-code-initiative/creedengo#-contribution)
 
 🤓 Main contributors
 --------------------
 
-check [ecoCode repository](https://github.com/green-code-initiative/ecoCode#-main-contributors)
+check [creedengo repository](https://github.com/green-code-initiative/creedengo#-main-contributors)
```

Comment:
please change it also ... because this repo will be the last one renamed, I think.